### ISSUE TITLE
White backgroud controls.

### ISSUE THREAD

**Issue by cpicanco** [BunsenLabs/bunsen1-dark-theme#3]
It is known issue with firefox.
In my case the theme does not work well with the Synaptic Package Manager filter white box.

**Comment by capn-damo** [comment]
I'm not seeing a problem with Synaptic (Installed: 0.81.3)

**Comment by cpicanco** [comment]
Hi,

v0.75.13 here

Please, if possible, make a quick test. Just write something on the "Quick filter". If it finds something or find no results, the background turns into white (sometimes yellow).
![screenshot - 02232015 - 05 21 09 pm](https://cloud.githubusercontent.com/assets/6511849/6336326/952f8002-bb80-11e4-8e1e-8633c415088a.png)

**Comment by capn-damo** [comment]
NB I'm running Debian sid, so it may be a gtk version issue?
No problem here as you can see (icons are different though?):

![screenshot](http://i.imgbox.com/rYqUgxVZ.jpg)

**Comment by cpicanco** [comment]
Which icon theme are you using?? I'm running Crunchbang 11 (RIP).

**Comment by capn-damo** [comment]
Using Crunchbang, Crunchbang-Dark, Faenza-Crunchbang gives the same result. It shouldn't affect anything else though. Did you set the gtk theme with "gksudo lxappearance"?

**Comment by cpicanco** [comment]
Yes. I am using a different icon theme, dã... Any Color Your Like

**Comment by cpicanco** [comment]
I dont know if it is an issue, but the libre office writer page is pretty
dark as well.

**Comment by cpicanco** [comment]
![screenshot - 02232015 - 08 54 03 pm](https://cloud.githubusercontent.com/assets/6511849/6340477/527130e4-bb9e-11e4-86b8-3792754939c9.png)

**Comment by capn-damo** [comment]
LO is fine for me. Is the theme over-riding your settings (Options -> Appearance -> Document background)?

**Comment by cpicanco** [comment]
No, it overides just the Automatic settings. And I think this is good. rsrsrr

**Comment by hhhorb** [comment]
Light-text-on-dark-background themes are problematic, any one I've ever used has issues. I think BlackMate works the best, but I haven't thouroughly tested it. Meld is another program that doesn't behave well. Really, it's easier just to temporarily change themes than to try and troubleshoot these issues.

For sure Bunsen1 should be the default theme, IMO, to avoid these problems.

**Comment by cpicanco** [comment]
I wound rather say adventurous than problematic. :)

**Comment by cpicanco** [comment]
Giving it a second thought, I think that Eye Healthy issues are related too. I have noticed that my eyes are not so dried as before. I fell that I am avoiding eye strain issues as well. And I am also using reduced back light monitor.
